test/bundler-integration/module/AccountRecovery.Module.specs.ts

```diff
@@ -195,10 +193,14 @@ describe("Account Recovery Module (via Bundler)", async () => {
     const recoveryRequestCallData = userSA.interface.encodeFunctionData(
       "execute",
       [
-        ecdsaModule.address,
+        accountRecoveryModule.address,
         ethers.utils.parseEther("0"),
-        ecdsaModule.interface.encodeFunctionData("transferOwnership", [
-          newOwner.address,
+        accountRecoveryModule.interface.encodeFunctionData("executeRecovery", [
+          ecdsaModule.address,
```

Comment:
can you make any transaction using executeRecovery or only to one of validation modules / just ECDSA?

Reply:
any transaction

Reply:
I see. this can be used to lock the smart account if we introduce such feature at implementation level. recently saw a guardian system that helps to lock the account (besides doing the recovery) [ can be helpful in the cases when your private key is stolen and not lost ]